### DISCUSSION TITLE
Bypass Bitbucket Approvals when using automerge

### DISCUSSION THREAD

**Discussion by adamxkabbara** [renovatebot/renovate#29927] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

None

### Please tell us more about your question or problem

Currently, I set bitbucket PRs to be able to merge if the PR build is passing, a few approvals, and tasks completed. I am using the automerge property renovate provides but the PR will not auto merge if it doesn't met those requirement. What I am trying to achieve: When I have renovates automerge on, I want to bypass approvals and tasks, but still what to have a passing build. Is there a way to set that up with renovate?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment, marked as answer]
There isn't such a setting currently. We either ignore all checks or all should pass.
